combine/signal_strength.sh

```diff
@@ -1,32 +1,56 @@
 #!/bin/bash
+set -e
+source utils/setup_cmssw.sh
+source utils/bashFunctionCollection.sh
 
 ERA=$1
 STXS_FIT=$2
-
-source utils/setup_cmssw.sh
-
+DATACARDDIR=$3
+CHANNEL=$4
+[[ -d $DATACARDDIR ]] || ( echo $DATACARDDIR is not a valid directory; exit 1 )
+if [[ $STXS_FIT == "inclusive" || $STXS_FIT == "stxs_stage0" ]]; then
+    STXS_SIGNALS=stxs_stage0
+elif [[ $STXS_FIT == "stxs_stage1p1" ]] ; then
+    STXS_SIGNALS=stxs_stage1p1
+else # Hesse default
+    STXS_SIGNALS=stxs_stage0
+fi
+WORKSPACE=$DATACARDDIR/${ERA}-${STXS_FIT}-workspace.root
+[[ -z $5 ]] && LOGFILE="output/log/signal-strength-$ERA-$CHANNEL-$STXS_FIT.log" || LOGFILE="output/log/signal-strength-$ERA-$5-$CHANNEL-$STXS_FIT.log"
+[[ -z $5 ]] && OUTPUTFILE="output/signalStrength/signal-strength-$CHANNEL-$STXS_FIT.txt" || OUTPUTFILE="output/signalStrength/signal-strength-$ERA-$5-$CHANNEL-$STXS_FIT.txt"
```

Comment:
Here also the folders output/signalStrength should be created if they do not exist